randovania/games/prime1/json_data/Phendrana Drifts.txt

```diff
@@ -741,6 +756,7 @@ Extra - asset_id: 1880625556
 > Fight Trigger; Heals? False
   > Event - Thardus
       All of the following:
+          Can Use Arm Cannon
```

Comment:
shouldn't this be "can shoot any beam"?

Reply:
I just universally replaced "Can Use Arm Cannon" with "Shoot Any Beam". Since you must spawn with 1 beam to do anything logically, I have reserved "Can Use Arm Cannon" for just factoring in templates